model_compression_toolkit/core/keras/default_framework_info.py

```diff
@@ -111,4 +123,4 @@
                                    ACTIVATION2MINMAX,
                                    LAYER2MINMAX,
                                    KERNEL_ATTRIBUTES,
-                                   OUTPUT_CHANNEL_INDEX)
```

Comment:
Please remove the variable OUTPUT_CHANNEL_INDEX if it is not being used anywhere else

Reply:
you are looking an old version. It was replaced by the dict variable.

Reply:
Isn't the actual old variable still exists (on line 118?), I see that it's being assigned there